.github/workflows/build.yml

```
@@ -0,0 +1,38 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.8"
+
+      - name: Cache SBT
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt', '**/plugins.sbt', '**/build.properties') }}
+
+      - name: Test
+        run: |
+          sbt "all clean" "all scalafmtSbtCheck scalafmtCheckAll" "all gatlingScalafixCheck" "test" "scripted"
```

I don't have much confidence in the parallel linting with the azure discs.
```suggestion
        run: sbt "clean;scalafmtSbtCheck;scalafmtCheckAll;gatlingScalafixCheck;test;scripted"
```
or maybe at least
```suggestion
        run: sbt "clean;all scalafmtSbtCheck scalafmtCheckAll;gatlingScalafixCheck;test;scripted"
```